drone-webdriver/src/main/java/org/jboss/arquillian/drone/webdriver/factory/remote/reusable/ReusableRemoteWebDriverToDestroy.java

```diff
@@ -0,0 +1,38 @@
+package org.jboss.arquillian.drone.webdriver.factory.remote.reusable;
+
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+import org.openqa.selenium.WebDriverException;
+import org.openqa.selenium.remote.RemoteWebDriver;
+
+/**
+ * @author <a href="mailto:mjobanek@redhat.com">Matous Jobanek</a>
+ */
+public class ReusableRemoteWebDriverToDestroy {
```

Comment:
Good point! I was thinking about the same. I decided to create a new class, because I wanted to use it only for this purpose and to limit confusions. I'm not sure what should be the content (if  I used this general class) in other cases https://github.com/arquillian/arquillian-extension-drone/blob/master/drone-webdriver/src/main/java/org/jboss/arquillian/drone/webdriver/factory/RemoteWebDriverFactory.java#L168-L189
If it should be removed when it is destroyed or add additional marker that it is destroyed.
There could be an addition question - why do we keep some kind of registry for `ReusableRemoteWebDriver`s and not for other drivers.